### PR TITLE
Redesign Team screen (Upcoming Match(es) and general layout)

### PR DESCRIPTION
**Pull request by Jarrydcrawford**
[ch245]

### CODE REVIEW

File: src/pages/Auth/Login.tsx [hunk 1]
```diff
@@ -11,6 +12,18 @@ import { Divider, Message } from 'semantic-ui-react';
...
+  color: #fff;
+`;
+
+const LoginForm = styled('form')`
+  color: #fff;
+`;
+
+const ForgotPasswordLink = styled(ButtonLink)`
+  color: #fff;
+`;
```
Comment (Jarrydcrawford) [inline src/pages/Auth/Login.tsx]: This kind of thing isn't ideal, would rather have a container, of some variety, which we use when the background is going to be dark.

File: src/components/MatchList.tsx [hunk 2]
```diff
@@ -16,30 +20,32 @@ interface Props {
...
-            </List.Description>
-          </List.Content>
-        </List.Item>
-      ))}
-    </List>
+    <>
+      <Heading>Matches</Heading>
+      <List divided={true} relaxed={true}>
+        {matches
+          .sort(compareMatchesByPlayedAt)
```
Comment (harrygr) [inline src/components/MatchList.tsx]: This mutates the matches list, which can cause issues as it comes from the apollo state. Ramda's `sort` returns a new list.

Comment (Jarrydcrawford) [reply src/components/MatchList.tsx]: Oh snap! Had no idea. I've had it too easy with pure array methods. I'll revert back.

File: src/components/MatchList.tsx [hunk 3]
```diff
@@ -16,30 +20,32 @@ interface Props {
...
+          .filter(match => !isMatchBeforeCurrentTime(match))
+          .map(match => (
+            <List.Item key={match.id}>
```
[the remark this answers is not in the document]
Comment (Jarrydcrawford) [reply src/components/MatchList.tsx]: Yep there's a ticket to complete remove this in place of the new Recent Match module so I'll leave it for then so we can get this moiged.

**Comment by Jarrydcrawford** [comment]
> There's a lot of emotion components that only set the color to white. Is there a cleaner approach perhaps that sets this colour in the parent. (say, in the Auth layout?)

Yeah ideally I'd want to have a light/dark container considering we have both styles on the design... 🤔

File: package.json [hunk 4]
```diff
@@ -40,7 +40,7 @@
     "timekeeper": "^2.1.2"
   },
   "scripts": {
-    "start": "react-scripts-ts start",
+    "start": "BROWSER=none react-scripts-ts start",
```
Comment (Jarrydcrawford) [inline package.json]: I will die on this hill.

**Pull request merged**